src/interfaces/Common.ts

```diff
@@ -1,5 +1,12 @@
+import { Log as _Log } from "@ethersproject/abstract-provider";
 import { BigNumber } from "../utils";
 
+export type Log = _Log & {
```

Comment:
It might be necessary to omit `topics` from `_Log` because it seems to be only present in ethers v5 Log, not viem Log. I think this should be OK but will need to test it.